Tests/RoutingKitTests/RouterTests.swift

```diff
@@ -127,4 +127,46 @@ final class RouterTests: XCTestCase {
         XCTAssertEqual(router.route(path: ["v1", "test", "foo"], parameters: &params), "b")
         XCTAssertEqual(router.route(path: ["v1", "foo"], parameters: &params), "c")
     }
+    
+    func testRouterDescription() throws {
+        // Use simple routing to eliminate the impact of registration order
+        let constA: PathComponent = "a"
+        let constOne: PathComponent = "1"
+        let paramOne: PathComponent = .parameter("1")
+        let anything: PathComponent = .anything
+        let catchall: PathComponent = .catchall
+        let router = TrieRouter<Int>()
+        router.register(0, at: [constA, anything])
+        router.register(1, at: [constA, constOne, catchall])
+        router.register(2, at: [constA, constOne, anything])
+        router.register(3, at: [anything, constA, paramOne])
+        router.register(4, at: [catchall])
+        // Manually build description
+        let desc = """
+→ \(constA)
```

Comment:
You can indent this normally and Swift will remove it automatically.

Reply:
Would it make sense to hardcode the description string here? That would help protect against regressions and also make it clearer what the expected output should be.

Reply:
Here’s my consideration:

`String->[PathComponent]` conversions have already been tested along with the router, and I added tests for `String->[PathComponent]->String` (`func testPathComponentDescription`) to ensure that `[PathComponent]->String` works well too. 

Using hard-coded string here is unnecessary because other tests have already tested the String expression. Avoiding hard-code makes the test dedicated for testing the description logic and algorithm.

Reply:
Fair enough 👍